composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "89580c52de91168aac8321460bd428e2",
+    "content-hash": "bc968c279762208bd4856bb6ebed42c3",
```

Comment:
Running `composer install` gives a warning about the lock file not being up to date:

```
└─[$]› ci
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. It is recommended that you run `composer update` or `composer update <package name>`.
```

I think it's because composer.lock was committed when the version was `*` before it was updated to `^1.0` in a later commit but the lock file wasn't updated or committed as well. Running this should update this hash and get rid of the warning.

```
composer update laravel-notification-channels/google-chat
```